test/box-luatest/gh_6530_net_box_opts_check_test.lua

```diff
@@ -144,7 +144,7 @@ g.test_netbox_all_methods_opts = function()
         stream:begin({}, 123)
     end)
     t.assert_error_msg_contains("options should be a table", function()
-        stream:commit(123)
+        stream:commit({}, 123)
```

Comment:
Since you made `stream:commit` accept old order of parameters, this test shouldn't require changing to work correctly.

Reply:
Fixed.